.github/workflows/release-please.yml

```diff
@@ -12,9 +12,6 @@ jobs:
             id-token: write
         steps:
             - uses: googleapis/release-please-action@v4
-              id: release
```

Comment:
I think we need to keep `id: release` to be able to reference it (e.g, `steps.release.outputs.release_created`) in the later steps.